CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 5.2.0
+- Improved typescript typings for `getHoldings` and `getInstitutionById`
+- Minor corrections to `README.md`
```

Comment:
In the changelogs, should we only include things that are changing about the API? 🤔 Wondering if readme changes are worth including here.

Reply:
true, weve done it in the past so i put it in, but i dont think we should going forward

Reply:
👍